HW1/P2/P2a.py

```diff
@@ -1,3 +1,30 @@
 from P2 import *
 
-# Your code here
+import findspark
+findspark.init('/Users/george/Documents/spark-1.5.0')
+import pyspark
+import matplotlib.pyplot as plt
+ 
+sc = pyspark.SparkContext()
+
+# crate rdd
+rdd = sc.parallelize(xrange(2000), 10)
+rdd = rdd.cartesian(rdd)
+
+# use formula given by pset
+rdd_map = rdd.map(lambda (i,j): ((i,j), (j/500.0-2, i/500.0-2)))
+
+# apply mandelbrot
+result = rdd_map.mapValues(lambda (x,y): mandelbrot(x,y))
```

Comment:
instead of two map steps, you could have applied Mandelblot in one step with rdd.map(lambda (i,j): (mandelbrot(j/500.0-2, i/500.0-2) ))